CHANGELOG.md

```diff
@@ -5,6 +5,45 @@
 
 ## Swift 6.0
 
+* [SE-0432][]:
+  Noncopyable enums can be pattern-matched with switches without consuming the
+  value you switch over:
+
+  ```swift
+  enum Lunch: ~Copyable {
+    case soup
+    case salad
+    case sandwich
+  }
+  
+  func isSoup(_ lunch: borrowing Lunch) -> Bool {
+    switch lunch {
+      case .soup: true
+      default: false
+    }
+  }
+  ```
+
+
+* [SE-0429][]:
+  Certain types that contain noncopyable fields, such as those without a deinit,
+  can now be consumed field-by-field:
```

Comment:
Worth making clear that it is the noncopyable fields of such aggregates that can be consumed individually.